packages/utils/handle-math-jax.js

```diff
@@ -0,0 +1,30 @@
+const handleMathJax = (rerun = false) => {
+    if (typeof window === 'undefined') {
+      return;
+    }
+  
+    const mathjaxScript = 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js';
```

Comment:
While this can be debated, why not add the js file by downloading it? A CDN is a good option, but maintaining support could be tricky if the owner of the file decides to change something drastically. 

Not everyone is aways keen on updating their fork.

Reply:
Thanks for the feedback @sirius93. The starter-kit is only a reference implementation of a frontend using our public APIs, designed to help you get started quickly without thinking about different aspects like CDNs, image hosting/resizing and so on. This is why we chose `cdn.jsdelivr.net` so that the script is served from an edge location. Also, once forked, we expect consumers to make significant changes to their fork to align it with their brand guidelines, tech, and so on. If they are using something like Vercel to deploy their fork (and therefore already have a CDN), they can always download this and self host.